cdk/lib/cdk-playground.ts

```diff
@@ -50,6 +53,16 @@ export class CdkPlayground extends GuStack {
 			imageRecipe: 'developerPlayground-arm64-java11',
 		});
 
+    // This is a temporary domain name for use in some ongoing Fastly Disaster Recovery testing
+    // It will be removed after this testing is complete
+    // todo - remove this Cname after Fastly disaster recovery testing is complete
+    new GuCname(this, 'FastlyDRTestDNS', {
+      app: ec2App,
+      ttl: Duration.hours(1),
+      domainName: fastlyDRTestDomainName,
+      resourceRecord: loadBalancer.loadBalancerDnsName,
```

Comment:
I think the CNAME needs to point to `dualstack.guardian.map.fastly.net.`, rather than the load balancer? The fastly service will then have the load balancer as an origin.

Reply:
Thanks, that is a really good point.
To set this, is it just a matter of setting resourceRecord to 'dualstack.guardian.map.fastly.net'? or is there more nuance involved?